src/components/tutor-input.cjsx

```diff
@@ -28,6 +31,19 @@ TutorInput = React.createClass
       <div className="floating-label">{@props.label}</div>
     </div>
 
+DayComponent = React.createClass
+  propTypes:
+    date:  React.PropTypes.string.isRequired
+    label: React.PropTypes.string.isRequired
+
+
+  render: ->
+    className = if moment(@props.date).startOf('day') < TimeStore.getNow()
+      'invalid'
+    else
+      'valid'
```

Comment:
Can you prefix this with `is-` (or the `invalid` case) and remove the other class name? I think that would also require adding a component name on it for the less file